.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "rdbg",
+            "name": "Minitest - current file",
+            "request": "launch",
+            "script": "-Itest ${file}",
+            "askParameters": false
```

Comment:
Should we ask for the specific test the person wants to run (the fixture name)? If we were trying to debug tests that use fixtures, we'd always end up running every single one, which is a very high number of tests.